core/config/config.go

```diff
@@ -65,6 +66,11 @@ func Load(ctx context.Context, db pg.DB, sdb *sinkdb.DB) (*Config, error) {
 	if err != nil {
 		return nil, errors.Wrap(err)
 	} else if ver.Exists() {
+		raftID := c.Id
+		if !idMatchesPG(ctx, raftID, db) {
+			raftDir := HomeDirFromEnvironment() + "/raft"
```

Comment:
might want to use [filepath.Join](https://godoc.org/path/filepath#Join) instead to ensure portability